lib/lost-utility.js

```diff
@@ -48,6 +62,49 @@ module.exports = function lostUtilityDecl(css) {
       );
     }
 
+    if (utility === 'overlay') {
+      var maxWidth = utilityArray[1] || '1024px',
+        numCols = utilityArray[2] || 12,
+        gutter = utilityArray[3] || '20px',
+        totalGutter = parseFloat(gutter) * (numCols - 1),
+        colWidth = (((parseFloat(maxWidth) - totalGutter) / numCols) / parseFloat(maxWidth)) * 100,
+        gutterPercentage = parseFloat(gutter) / parseFloat(maxWidth) * 100,
+        position = 0,
+        gradient = 'to right, ';
+
+      color = utilityArray[4] || '#e6f6ff';
+
+      if(!unitsMatch(maxWidth, gutter)) {
+        throw decl.error('lost-utility: Please use the same units for width and gutter.', { plugin: 'lost', word: 'lost-utility' });
```

Comment:
Not for here, but we should make this a method we can more easily access anywhere–that way we can ensure consistency.